test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,96 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
```

Comment:
Use override_qengines to be consistent?

Reply:
I changed to override_qeninges, calibrate to test_only_eval for all API in another PR (#49266). I can update it in this PR as well